glossary.yml

```
@@ -990,9 +1008,16 @@
   en:
     term: "constructor"
     def: >
-      A function that creates an [object](#object) of a particular [class](#class). In
-      the [S3](#s3) object system, constructors are a convention rather than a requirement.
-
+      A function that creates an [object](#object) of a particular
+      [class](#class). In the [S3](#s3) object system, constructors are a
+      convention rather than a requirement.
+  es: 
+    term: "constructor"
+    def: >
+      Una función que crea un [objeto](#object) de una [clase](#class)
+      particular. En el sistema de objetos de [S3](#s3), los constructores
+      son más una convención que un requerimiento.
```

Creo que aquí deberíamos de usar "requisito" (circunstancia o condición necesaria para algo) y no "requirimiento" (la acción de requerir)
```suggestion
      son más una convención que un requisito.
```